app/views/posts/show.html.erb

```diff
@@ -1,29 +1,165 @@
-<h2><%= @post.title %></h2>
+<style>
+  /* Style for the post details section */
+  .post-details {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    border-bottom: 1px solid #ccc;
+    padding-bottom: 10px;
+  }
 
-<div class="post-details">
-  <div class="post-author">
-    <h4><%= @post.author.name %></h4>
-  </div>
-  <div class="post-metadata">
-    <p><%= @post.created_at.strftime("%b %d, %Y") %></p>
-    <p>Number of comments: <%= @post.comments.count %></p>
-    <p>Number of likes: <%= @post.likes_counter %></p>
-  </div>
-</div>
+  .post-author {
+    display: flex;
+    align-items: center;
+  }
+
+  .post-author h4 {
+    margin-left: 10px;
+  }
+
+  .post-metadata {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+  }
+
+  .post-metadata p {
+    margin: 5px 0;
+  }
+
+  /* Style for the comments section */
+  h3 {
+    margin-top: 30px;
+    margin-bottom: 20px;
+  }
+
+  ul li.media {
+    border: 1px solid #ccc;
+    padding: 10px;
+  }
+
+  img.rounded-circle {
+    border-radius: 50%;
+  }
+
+  button.add-post {
+    margin-top: 20px;
+    padding: 10px 20px;
+    background-color: #007bff;
+    color: #fff;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+  }
+
+  button.add-post:hover {
+    background-color: #0069d9;
+  }
+
+  /* Style for the body */
+  body {
+    font-family: Arial, sans-serif;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #333;
+    background-color: #f8f9fa;
+  }
+
+  /* Style for the container */
+  .container {
+    max-width: 960px;
+    margin: 0 auto;
+    padding: 0 15px;
+  }
 
-<p><%= @post.text %></p>
+  /* Style for the row */
+  .row {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: -15px;
+    margin-left: -15px;
+  }
 
-<hr>
+  /* Style for the columns */
+  .col-md-8 {
+    flex: 0 0 66.666667%;
+    max-width: 66.666667%;
+    position: relative;
+    width: 100%;
+    padding-right: 15px;
+    padding-left: 15px;
+  }
 
-<h3>Comments:</h3>
+  /* Style for the headings */
+  h2, h3, h4, h5, h6 {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-weight: bold;
+    line-height: 1.2;
+    margin-top: 0;
+    margin-bottom: 0.5rem;
+  }
 
-<ul>
-  <% @post.comments.each do |comment| %>
-    <li>
-      <p><strong><%= comment.user.name %>:</strong> <%= comment.text %></p>
-    </li>
-  <% end %>
-</ul>
+  /* Style for the paragraph */
+  p {
+    margin-top: 0;
+    margin-bottom: 1rem;
+  }
+
+  /* Style for the hr */
+  hr {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    border: 0;
+    border-top: 1px solid #ccc;
+  }
+</style>
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job adding all the changes recommended by the previous reviewer. However I think it would be better if you move all your styling code in a specific styling file inside your `app/assets/stylesheets/` directory. This for best practices reasons.